tutoraspects/asset_command_helpers.py

```diff
@@ -474,19 +483,20 @@ def check_asset_names(echo):
     warn = 0
 
     names = set()
-    for file_name, asset in _get_asset_files():
+    for _, asset in _get_asset_files():
         for k in ("slice_name", "dashboard_title", "database_name"):
             if k in asset:
                 if asset[k] in names:
                     warn += 1
                     echo(
-                        f"WARNING: Duplicate name {asset[k]} in {file_name}, this "
-                        f"could confuse users, consider changing it."
+                        f"WARNING: Duplicate {k} {asset[k]} in {asset.get('_file_name')}"
                     )
                 names.add(asset[k])
                 break
 
-    echo(f"{warn} duplicate names detected.")
+    echo(
+        f"{warn} duplicate names detected. This could confuse users, consider changing them."
```

Comment:
I'm not sure how useful these warnings are these days, IIRC we had intentionally duplicated chart names between the individual learner and at-risk learner dashboards so maybe we should just get rid of this check?

Reply:
That's fair, I'll remove it!

Reply:
oops I didn't see your comment @Ian2012 - it would still throw a warning for something like the course comparison dashboard where there are multiple charts named the same thing (but one is on course level and one is on course run level) There's nothing wrong/confusing about that, though, so I don't think we need the warning anymore.